integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -369,7 +369,8 @@ def test_mod_pmod_by_zero(data_gen, overflow_exp):
 def test_cast_neg_to_decimal_err():
     # -12 cannot be represented as decimal(7,7)
     data_gen = _decimal_gen_7_7
-    if is_before_spark_314() or ((not is_before_spark_320()) and is_before_spark_322()):
+    #if is_before_spark_313() or ((not is_before_spark_320()) and is_before_spark_322()):
```

Comment:
why do we need this comment?

Reply:
My bad. Forgot to remove it after testing.